docs/roadmap/index.md

```diff
@@ -93,7 +93,7 @@ FPGA and MCU
 TPM stack and writing responses back to FIFO
 - review and update existing documentation, add entry to changelog
 
-State: Backlog
+State: In progress
 
 ## 6. Base tests
```

Comment:
@krystian-hebel Please change the status of phase 6 and phase 8 to "in progress".

Reply:
Done